github/commitstatus.go

```diff
@@ -96,14 +102,48 @@ func (s CommitStatus) Add(sha, state, targetURL, description string) error {
 	req.Header.Set("Accept", "application/vnd.github.v3+json")
 	req.Header.Set("Content-Type", "application/json")
 
+	var response httpResponse
+	var retry bool
+	timeToSleep := time.Second * 0
```

Comment:
The constants in `time` are to be considered as unit of measure, so it is customary to write the constant on the right of the multiplication:

```go
	timeToSleep := 0 * time.Second // 0 seconds
```

Reply:
Changed as suggested.